src/shared/components/palette/palette.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable, Type } from '@angular/core';
 import { CaseField } from '../../domain/definition/case-field.model';
 import { WriteAddressFieldComponent } from './address/write-address-field.component';
+import { CaseFileViewFieldComponent } from './case-file-view';
```

Comment:
@johnbenjamin-hmcts - probably better to use the fully qualified form here, i.e. `'./case-file-view/case-file-view-field.component'`

Reply:
thank you @Daniel-Lam, I have now done the changes.